server/controller/src/main/java/ai/starwhale/mlops/domain/job/cache/HotJobHolder.java

```diff
@@ -27,13 +27,17 @@
  */
 public interface HotJobHolder {
 
-    void adopt(Job job);
+    void add(Job job);
 
-    Collection<Job> ofIds(Collection<Long> ids);
+    Collection<Job> getJobs(Set<Long> ids);
```

Comment:
what is the benefit of renaming this method

Reply:
The form ofxxx has been uniformly used